### DISCUSSION TITLE
Case V2 assembly

### DISCUSSION THREAD

**Discussion by rjbutler** [doudar/SmartSpin2k#251] [Q&A] [answered]
Are there instructions or pictures showing how to assemble Case V2?  I'm in the process of printing it and my PCB should arrive tomorrow, so maybe once I have it all in front of me it will be obvious, but if there are some pictures it might be helpful.

Thanks

**Comment by eMadman** [comment, marked as answer]
I just did an assembly - it's very straightforward and everything really will only have "one spot" to go.  I don't have mine open on my desk right now but can take pictures if you're totally stuck. 

I forget what length wood screws I used, but if you prefer heat set inserts and smaller machine screws, I made a [fork of the repo and modded ](https://github.com/eMadman/SmartSpin2k/tree/develop/Hardware/MODS/Case%20V2%20-%20Heat%20Set%20Inserts)the case to accept the inserts instead.  Haven't had a chance to print and test it myself but if it's of interest, I'd welcome the feedback

you'll want good tolerances for the 11t gear.  I was finding the inner radius too big and it was impossible to put the gear on the motor shaft.... turned out my perimeters were too wide.  

You'll also want a few dabs of superglue for the cup and where the cup/shaft mate, but it should all be obvious during assembly.  Also suggest using a bit of grease on the gear teeth... I used EP2, but I think white lithium or dielectric will also do here.

**Reply by rjbutler** [reply]
@eMadman Thanks, I noticed the same with the 11t gear, although I can get it started on the motor so maybe it's OK. 

I noticed you posted some photos for an alternate mounting arm for the IC4.  Have you printed that yet? I assume with the current arm you just use a rubber band or something to attach it to the handlebar post?

**Reply by eMadman** [reply]
Correct - a rubber band or velcro strap will loosely secure it to the bike.  Because the knob is round and there's no hard mount for the ss2k, it tends to tip around as it turns the knob.  It won't fall off, but it'd be nice to have it more steady.  I decided to try my hand at a fixed mount since my use case is only for use at home.  Mount is designed about as far as I can get without printing and test fitting it and adjusting the length of the arm.  I'm hoping to use a slot fit for the bike side of it so that it can dock in place and then lift the whole thing off when i'm done.

**Reply by rjbutler** [reply]
@eMadman @doudar  I'm not sure how the stepper motor should fit.  I ordered the one from the parts list, but it actually doesn't match the brand in the instructables article and it looks like the case is designed for a 1.5" housing rather than the 1.2" the link takes me to.  I guess I can try to modify the case to fit (not something I've done before) or print a shim to make it fit in the current case.  Or order another stepper motor.

Unless I'm missing something?

**Reply by eMadman** [reply]
I grabbed a random Amazon stepper which has a ~16-17mm wide JST-looking connector that got in the way of assembly.  Since you've already printed, here's the physical mods I made in order to get my motor/gear to fit.  What you're seeing is my in-progress Schwinn IC4 mod that applies some of those learnings already.  
![image](https://user-images.githubusercontent.com/24726844/143040546-93bf305c-a1d9-47f5-8a78-08f883540690.png)

Motor is a standard nema 17... should be approx 42mm square, 37mm long (not counting shaft).  For our purposes, any nema 17 will do.  Looking at the Amazon link, it is indeed a smaller pancake motor.  It'll work, but as you mentioned, it will need a shim to fill the gap.  I [grabbed these ](https://www.amazon.ca/Twotrees-Nema17-Stepper-17HS4401S-Printer/dp/B08C2KDY6R/ref=sr_1_5?keywords=nema%2B17&qid=1637677288&sr=8-5&th=1 )because they were in-stock and reasonably priced.
